src/frontend/screens/Settings/Experiments/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { useNavigation } from "react-navigation-hooks";
+import { NavigationStackScreenComponent } from "react-navigation-stack";
+import SettingsContext from "../../../context/SettingsContext";
+import HeaderTitle from "../../../sharedComponents/HeaderTitle";
+import {
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+} from "../../../sharedComponents/List";
+
+const m = defineMessages({
+  p2pUpgrades: {
+    id: "screens.Settings.Experiments.p2pUpgrades",
+    defaultMessage: "P2P upgrades",
+    description: "Label for p2p upgrades",
+  },
+  p2pUpgradesDesc: {
+    id: "screens.Settings.Experiments.p2pUpgradesDesc",
+    defaultMessage: "Share and receive updates with nearby devices",
+    description: "Label for p2p upgrades description",
+  },
+  active: {
+    id: "screens.Settings.Experiments.active",
+    defaultMessage: "Active",
+    description: "Inidicates whether directional arrow is active on map",
+  },
+  inactive: {
+    id: "screens.Settings.Experiments.inactive",
+    defaultMessage: "Inactive",
+    description: "Inidicates whether directional arrow is NOT active on map",
+  },
+  directionalArrow: {
+    id: "screens.Settings.Experiments.directionalArrow",
+    defaultMessage: "Directional Arrow",
+  },
+  title: {
+    id: "screens.Settings.Experiments.title",
+    defaultMessage: "Experiments",
+  },
+});
+
+const Experiments: NavigationStackScreenComponent = () => {
+  const [{ experiments, directionalArrow }, setSettings] = React.useContext(
+    SettingsContext
+  );
+  const { navigate } = useNavigation();
+
+  return (
+    <List testID="experimentsList">
+      <ListItem
+        onPress={() => navigate("P2pUpgrade")}
+        testID="p2pUpgradeExperimentButton"
+      >
+        <ListItemIcon iconName="sync" />
```

Comment:
is the gif updated to reflect most recent changes? seems like it's not detecting this icon correctly, although it seems to be a valid icon